plugins/dubbo/filter/graceful_shutdown/server_intercepter.go

```diff
@@ -0,0 +1,58 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// nolint
+package graceful_shutdown
+
+import (
+	"dubbo.apache.org/dubbo-go/v3/protocol"
+
+	"github.com/apache/skywalking-go/plugins/core/operator"
+	"github.com/apache/skywalking-go/plugins/core/tracing"
+)
+
+type ServerInterceptor struct {
+}
+
+func (c *ServerInterceptor) BeforeInvoke(invocation operator.Invocation) error {
+	filterInvoker := invocation.Args()[1].(protocol.Invoker)
+	dubboInv := invocation.Args()[2].(protocol.Invocation)
+	url := filterInvoker.GetURL()
+	if url == nil {
+		return nil
+	}
+	s, err := tracing.CreateEntrySpan(generateOperationName(filterInvoker, dubboInv), func(k string) (string, error) {
+		attachment, _ := dubboInv.GetAttachment(k)
+		return attachment, nil
+	}, tracing.WithLayer(tracing.SpanLayerRPCFramework),
+		tracing.WithTag(tracing.TagURL, url.String()),
+		tracing.WithComponent(3))
```

Comment:
About this ID, should we consider some management/registration mechanism in the go? I know these are registered at the backend, at least, is this registration mentioned in go plugin dev doc?

Reply:
I could add a section to help developers find the component ID. 
I think we don't need the management of the component ID registration in the go agent.